### PR TITLE
assingment 1

### PR DESCRIPTION
**Pull request by nmt-hitu**

### CODE REVIEW

File: week1/main.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	// "errors"
```
Comment (havinhdat) [inline week1/main.go:9]: should remove useless comments

File: week1/main.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+func main() {
+
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Print(">")
+	for scanner.Scan(){
+		text := scanner.Text()
+		if text != "" {
+			fmt.Print(process(text))
+			}
+		fmt.Print("\n >")
```
Comment (havinhdat) [inline week1/main.go:21]: Above Print doesn't have space before `>`, so it's best to define `>` as a constant

File: week1/main.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	}
+
+}
+
+func process(text string) string{
```
Comment (havinhdat) [inline week1/main.go:26]: function name could be named better, I personally don't expect an output coming out of a `process` function

File: week1/main.go [hunk 4]
```diff
@@ -0,0 +1,64 @@
+package main
```
Comment (havinhdat) [inline week1/main.go:1]: Can run `go fmt` to clean up the code (indent, spaces ...)
https://blog.golang.org/go-fmt-your-code

File: week1/main.go [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+	
+	b, error := strconv.ParseFloat(expr[2], 10);
+
+	if error !=nil {
+		return "b has to a number"
+	}	
+
+	op := expr[1];
+
+	var result float64
```
Comment (hieuvo) [inline week1/main.go:43]: the result should have same format as inputs, ex: `2 + 3 = 5.000` is weird

File: week1/main.go [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+			fmt.Print(process(text))
+			}
+		fmt.Print("\n >")
+	}
+
+}
+
+func process(text string) string{
+
+	expr := strings.Split(text, " ");
```
Comment (hieuvo) [inline week1/main.go:28]: better if we can handle the input with multiple spaces: ex `2  + 3`

**Pull request merged**